frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card.tsx

```diff
@@ -58,6 +60,16 @@ const ObjectDashboardBucketsCard: React.FC<DashboardItemProps> = ({
     BucketsCardQueries.UNHEALTHY_BUCKETS_CLAIMS,
     'result',
   ]);
+  const bucketsLinksResponse = prometheusResults.getIn([
+    BucketsCardQueries.BUCKETS_LINK_QUERY,
+    'result',
+  ]);
+
+  const noobaaSystemAddress = getMetric(bucketsLinksResponse, 'system_address');
+  const noobaaSystemName = getMetric(bucketsLinksResponse, 'system_name');
+  let link = '';
+  if (!_.isNil(noobaaSystemAddress) && !_.isNil(noobaaSystemName))
+    link = `${noobaaSystemAddress}/fe/systems/${noobaaSystemName}/buckets/data-buckets`;
```

Comment:
can you give an example of what this link looks like fully resolved ?

Reply:
https://jenia.us-east-2.elb.amazonaws.com:8443/fe/systems/noobaa/buckets/data-buckets

Reply:
is this an external link ? If yes ,than we should use `a href` instead